Code/GraphMol/FileParsers/MolFileWriter.cpp

```diff
@@ -800,27 +800,60 @@ void GetMolFileBondStereoInfo(const Bond *bond, const INT_MAP_INT &wedgeBonds,
             ROMol::OEDGE_ITER beg, end;
             boost::tie(beg, end) =
                 bond->getOwningMol().getAtomBonds(bond->getBeginAtom());
+            bool atomHasDuplicateRanks = false;
```

Comment:
I'm being picky here, but since it's not the same atom that has duplicate ranks, I'd name this `nbrsHaveDuplicateRanks`.